<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1044</width>
      <height>516</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="3">
      <widget class="QRadioButton" name="orRadioButton">
       <property name="text">
        <string>OR</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7" colspan="2">
      <widget class="QPushButton" name="addConditionButton">
       <property name="text">
        <string>Add condition</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="removeFromDBButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QRadioButton" name="andRadioButton">
       <property name="text">
        <string>AND</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7" colspan="2">
      <widget class="QPushButton" name="searchByQueryButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QPushButton" name="addToConditionsComboBoxButton">
       <property name="text">
        <string>-&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3" colspan="3">
      <widget class="QListWidget" name="allConditionsListWidget"/>
     </item>
     <item row="0" column="2" rowspan="8">
      <widget class="QTableWidget" name="resultTableWidget"/>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="AddToDBButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="8" colspan="2">
      <widget class="QTableWidget" name="mainTableWidget"/>
     </item>
     <item row="9" column="0" colspan="3">
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6" colspan="3">
      <widget class="QListWidget" name="resultConditionsListWidget"/>
     </item>
     <item row="7" column="7" colspan="2">
      <widget class="QPushButton" name="searchByConditionsButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="6">
      <widget class="QTextEdit" name="queryTextWidget">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="removeFromDBButton2">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="2">
      <widget class="QComboBox" name="selectColumnComboBox"/>
     </item>
     <item row="3" column="5">
      <widget class="QComboBox" name="selectOperatorComboBox">
       <item>
        <property name="text">
         <string>=</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&gt;=</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;=</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QLineEdit" name="selectValueLine">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="6">
      <widget class="QPushButton" name="removeFromResultConditionsButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item row="8" column="6" colspan="3">
      <widget class="QPushButton" name="showQueryButton">
       <property name="text">
        <string> Show query</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="clearQueryTextButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1075</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
